### PR TITLE
Adding Avax Vault (disabled)

### PR DESCRIPTION
**Pull request by fqlx**
- Adding the Avax Vault to the UI.
- Will add the network switcher in a different MR.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,8 @@
+REACT_APP_VERCEL_GIT_COMMIT_REF=staging # used to track which branch is used to deploy the app. 'staging' uses mainnet for URLs
```
Comment (kenchangh) [inline .env.example:1]: This file should already be present in `.env.sample` so we can delete it

File: shared/src/designSystem/colors.ts [hunk 2]
```diff
@@ -44,6 +44,7 @@ const colors = {
     yvUSDC: "#3E73C4",
     stETH: "#00A3FF",
     AAVE: "#2EBAC6",
+    WAVAX: "#627EEA", // FIXME: Avax
```
Comment (kenchangh) [inline shared/src/designSystem/colors.ts:47]: Get the color hex from Vinnie so we can finalize this

File: .env.example (hunk 1 above)
Comment (fqlx) [reply .env.example]: Deleted!

File: shared/src/designSystem/colors.ts (hunk 2 above)
Comment (fqlx) [reply shared/src/designSystem/colors.ts]: Fixed!

**Review by kenchangh — APPROVED** [review]

**Pull request merged**